api/envoy/extensions/filters/network/mysql_proxy/v3/mysql_proxy.proto

```diff
@@ -15,15 +18,71 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // MySQL Proxy :ref:`configuration overview <config_network_filters_mysql_proxy>`.
 // [#extension: envoy.filters.network.mysql_proxy]
 
+// Auth information of MySQL.
+message AuthInfo {
+  config.core.v3.DataSource auth_username = 1;
```

Comment:
Probably just put the sensitive annotations on the fields in here directly to factor them out.

Reply:
fixed.